thirdparty/thorvg/src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -709,7 +709,7 @@ static bool _toColor(const char* str, uint8_t* r, uint8_t* g, uint8_t* b, char**
         *ref = _idFromUrl((const char*)(str + 3));
         return true;
     } else if (len >= 10 && (str[0] == 'h' || str[0] == 'H') && (str[1] == 's' || str[1] == 'S') && (str[2] == 'l' || str[2] == 'L') && str[3] == '(' && str[len - 1] == ')') {
-        float th, ts, tb;
+        float_t th, ts, tb;
```

Comment:
This broke the 32-bit builds again, fixed it up with 38716b94dbcc9dbfb60a49399c5118f01fc79c45.

There's already a patch merged upstream to fix this, but if it doesn't make it to the next 0.13.x, we need to make sure to reapply the patch downstream.

Reply:
> This broke the 32-bit builds again, fixed it up with [38716b9](https://github.com/godotengine/godot/commit/38716b94dbcc9dbfb60a49399c5118f01fc79c45).
> 
> There's already a patch merged upstream to fix this, but if it doesn't make it to the next 0.13.x, we need to make sure to reapply the patch downstream.

Could a Godot CI check be implemented for this issue so that everyone is alerted when similar ones occur?

Reply:
Technically yes, but it's a matter of tradeoff between having a reasonably wide coverage on CI, and the amount of resources and time it takes for CI to pass on every single PR update.

Adding a 32-bit Linux or Windows build could be worth doing, but that's one more build to wait for many times a day.

Reply:
I'm surprised WebAssembly builds don't catch this, as WebAssembly is still 32-bit in its base specification.

Reply:
I'm not sure if this is possible in github - we run an extended text suite daily, the daemon figures out who is to blame and sends them an alert.